File: src/main/java/me/arrush/javabase/databases/MySQLDatabase.java
package me.arrush.javabase.databases;

import java.sql.SQLException;

@SuppressWarnings("WeakerAccess")
public final class MySQLDatabase extends Database {
    MySQLDatabase(String ip, int port, String dbName) throws ClassNotFoundException, SQLException {
        super(dbName, DatabaseType.MYSQL, "com.mysql.jdbc.Driver", getMySQLConnection(ip, port, dbName));
    }
}

File: src/main/java/com/arrushc/javabase/entities/database/MySQLDatabase.kt
package com.arrushc.javabase.entities.database

import com.arrushc.javabase.entities.Database

class MySQLDatabase internal constructor(ip: String?, port: Int, dbName: String?) : Database(dbName!!, DatabaseType.MYSQL, "com.mysql.jdbc.Driver", getMySQLDataSource(ip, port, dbName))